### ISSUE TITLE
gives issue syntex error

### ISSUE THREAD

**Issue by kashif-sol** [pgrimaud/instagram-user-feed#335]
**Version(s) affected**: 16

![image](https://user-images.githubusercontent.com/30724786/187451356-2e22e1d9-99c7-40f3-be21-6b1b56e50f22.png)

**Comment by kashif-sol** [comment]
anyone here using package and face these issues

**Comment by fnuppy** [comment]
Hello,
I have the same

**Comment by schelmo** [comment]
i stumbeld on this also, while trying to get a user profile
```php
$api->getProfile($username);
```
the error is thrown in https://github.com/pgrimaud/instagram-user-feed/blob/8ae164be0c2849befa2d3e6d14317bf54652e7f0/src/Instagram/Transport/JsonProfileDataFeedV2.php#L28
the Problem is in 
https://github.com/pgrimaud/instagram-user-feed/blob/8ae164be0c2849befa2d3e6d14317bf54652e7f0/src/Instagram/Transport/AbstractDataFeed.php#L56-L64
instagram is giving an html file back
so the json couldnt be decoded ( Syntax Error )

but i couldn't determine what the real problem is. i couldnt find an error in the html response

on my local application (different ip) i dont have the problem

**Comment by fnuppy** [comment]
Possible pb with permissions on server and files can not be generated?

**Comment by schelmo** [comment]
> Possible pb with permissions on server and files can not be generated?

nope, its a problem with instagram, since its giving a html file back instead of a json. 
seems my local cache file has the proper cookies set

**Comment by kashif-sol** [comment]
i think problem is Instagram ask for login

**Comment by Rubberg** [comment]
Same pb here!

**Comment by IrfanPatelDev** [comment]
I am facing the same issue, Please someone help with this issue
